sparkle/platform/settings_objects.py

```diff
@@ -906,6 +1035,232 @@ def get_smac2_max_iterations(self: Settings) -> int | None:
         max_iterations = self.__settings["smac2"]["max_iterations"]
         return int(max_iterations) if max_iterations.isdigit() else None
 
+    # Configuration: SMAC3 specific settings ###
```

Comment:
This approach looks not very maintainable for future integrations of other AAC/HPO frameworks into Sparkle. Make ticket to make this process easier/better maintainable.

Reply:
Agreed, but we should I think in general discuss an overhaul of the settings class as it is too much code to maintain a single setting